### ISSUE TITLE
Deprecated selector in `block-cursor/styles/block-cursor.less`

### ISSUE THREAD

**Issue by walter-0** [olmokramer/atom-block-cursor#33]
In `block-cursor/styles/block-cursor.less`: 

Starting from Atom v1.13.0, the contents of `atom-text-editor` elements are no longer encapsulated within a shadow DOM boundary. This means you should stop using `:host` and `::shadow` pseudo-selectors, and prepend all your syntax selectors with `syntax--`. To prevent breakage with existing style sheets, Atom will automatically upgrade the following selectors:

* `atom-text-editor::shadow .cursors .cursor` => `atom-text-editor.editor .cursors .cursor`

* `atom-text-editor[mini]:not(.is-focused)::shadow .cursors .cursor` => `atom-text-editor[mini]:not(.is-focused).editor .cursors .cursor`

Automatic translation of selectors will be removed in a few release cycles to minimize startup time. Please, make sure to upgrade the above selectors as soon as possible.

**Comment by 2114L3** [comment]
also have this issue. 
archlinux 4.8.13-1-ARCH
atom 1.13.0 x64

**Comment by CamelKing** [comment]
+1

**Comment by dschetel** [comment]
+1

**Comment by ghost** [comment]
I've replaced _::shadow_ by _.editor_, saved _block-cursor.less_ and restarted atom. Still no cursor.

**block-cursor.less**
```
atom-text-editor.editor .cursors .cursor {
  opacity: 1;
  color: transparent;
  background-color: transparent;
  border: none;
  transition-property: background-color, border-color, opacity;
}

atom-text-editor[mini]:not(.is-focused).editor .cursors .cursor {
  display: none;
}

```

**Atom 1.15.0 x64**

**Comment by adamrenklint** [comment]
Published v0.13.2 that resolves this issue
